src/OrleansRuntime/Hosting/DefaultSiloServices.cs

```diff
@@ -55,6 +55,7 @@ internal static void AddDefaultServices(IServiceCollection services)
             services.TryAddFromExisting<IMessagingConfiguration, GlobalConfiguration>();
             services.TryAddFromExisting<ITraceConfiguration, NodeConfiguration>();
 
+            services.AddLogging();
```

Comment:
technically this isn't needed nor used in this PR, right?

Reply:
that's right. I can remove it